tests/test_utilities.py

```diff
@@ -1,4 +1,4 @@
-""" test pyvista.utilities """
+""" test pyvista utilities """
```

Comment:
```suggestion
"""Test pyvista utilities."""
```
Or since we don't seem to be including other utilities,
```suggestion
"""Test pyvista core utilities."""
```